python/nammu/test/test_nammu.py

```diff
@@ -412,3 +412,81 @@ def test_redo_arabic_pane(self, arabic, nammu):
+        # Split: only toggling in the same direction should be enabled.
+        view.toggle_split(orientation)
+        assert view.container.getOrientation() == orientation
+        assert toggleThis.isEnabled()
+        assert not toggleOrthogonal.isEnabled()
+        assert not toggleArabic.isEnabled()
+        # Try toggling orthogonal split: nothing should happen.
+        view.toggle_split(getattr(JSplitPane,
+                                  "{}_SPLIT".format(orthogonal.upper())))
+        assert view.container.getOrientation() == orientation
+        assert toggleThis.isEnabled()
+        assert not toggleOrthogonal.isEnabled()
+        assert not toggleArabic.isEnabled()
+        # Toggle again: no split view.
+        view.toggle_split(orientation)
+        assert isinstance(view.container, JScrollPane)
+        assert toggleThis.isEnabled()
+        assert toggleOrthogonal.isEnabled()
+        assert toggleArabic.isEnabled()
+
+    def test_toggling_arabic_split(self, nammu):
+        """
+        Test toggling split views in Arabic file.
+        """
+        controller = nammu.atfAreaController
+        view = controller.view
+        menuView = controller.controller.menuController.view
+        toggleVertical = menuView.getMenuItemByName(
+            "Window", "Toggle Vertical Split Editor")
+        toggleHorizontal = menuView.getMenuItemByName(
+            "Window", "Toggle Horizontal Split Editor")
+        toggleArabic = menuView.getMenuItemByName(
+            "Window", "Toggle Arabic Translation Editor")
+        # Split with Arabic pane but with no Arabic text: it creates the
+        # vertical split view.  Only toggling Arabic should be enabled.
+        view.toggle_split_arabic(JSplitPane.VERTICAL_SPLIT, "", "")
+        assert view.container.getOrientation() == JSplitPane.VERTICAL_SPLIT
+        assert not toggleVertical.isEnabled()
+        assert not toggleHorizontal.isEnabled()
+        assert toggleArabic.isEnabled()
+        # Try toggling horizontally: nothing should happen.
+        view.toggle_split(JSplitPane.HORIZONTAL_SPLIT)
+        assert view.container.getOrientation() == JSplitPane.VERTICAL_SPLIT
+        assert not toggleVertical.isEnabled()
+        assert not toggleHorizontal.isEnabled()
+        assert toggleArabic.isEnabled()
+        # Toggle Arabic pane: restore view with no split.
+        view.toggle_split_arabic(JSplitPane.VERTICAL_SPLIT, "", "")
+        assert isinstance(view.container, JScrollPane)
+        assert toggleVertical.isEnabled()
+        assert toggleHorizontal.isEnabled()
+        assert toggleArabic.isEnabled()
+        # Full Arabic mode: it shouldn't be possible to toggle anything, we
+        # want to always keep the Arabic pane.
+        nammu.arabic()
+        assert view.container.getOrientation() == JSplitPane.VERTICAL_SPLIT
+        assert not toggleVertical.isEnabled()
+        assert not toggleHorizontal.isEnabled()
+        assert not toggleArabic.isEnabled()
```

Comment:
TODO: Add test that when there is an arabic file and toggling vert/horiz won't make disappear the arabic pane.